xml/net_dns.xml

```diff
@@ -856,12 +856,12 @@ too easy to do. -->
 };
 
 zone "localhost" in {
-       type master;
+       type primary;
```

Comment:
It is snippet of a configuration file for the dns server. So, this is wrong as far as the dns service has not implemented new keyword (i don't know). 

However, talking about yast, yast exports old ```master``` keyword into the config.